roles/pulp_common/tasks/install_pip.yml

```diff
@@ -300,6 +300,9 @@
       when: item.value.source_dir is defined
       environment:
         SETUPTOOLS_USE_DISTUTILS: stdlib
+        COMPOSE_PROFILE: standalone
+        DEV_SOURCE_PATH: 'pulpcore:pulp_ansible:pulp_container:galaxy_ng:galaxy-importer'
+        LOCK_REQUIREMENTS: 0
```

Comment:
maybe add a comment explaining why those vars exists? might be confusing to see `COMPOSE_PROFILE` here

Reply:
I was thinking more about changing galaxy_ng to not require this var

Reply:
@fao89 agreed, lets make galaxy setup.py to look only to DEV_SOURCE_PATH and LOCK_REQUIREMENTS